### ISSUE TITLE
方便透露一下数据的来源吗？

### ISSUE THREAD

**Issue by MashiMaroLjc** [PaddlePaddle/Knover#14]
数据不开源我可以理解，但透露一下数据的来源没问题吧？

论文只是简单的提了一下中文数据来自中文的社交媒体，能否具体一点呢？
微博，豆瓣小组，还是百度贴吧？

不同的来源上文谈话的内容风格和话题差异还是比较大的，希望可以提供一下。
谢谢

**Comment by portia1026** [comment]
> 不同的来源上文谈话的内容风格和话题差异还是比较大的
同意。为了保证训练语料的多样性和覆盖面，PLATO-2训练使用的中文数据来源是很丰富的（并不局限在一个数据集上），包括多个公开数据集，脱敏的社交媒体数据，以及人工标注数据。

**Comment by DvHuang** [comment]
您好，感谢开源，能说一下各类数据集的大概比例吗？
